Project_CA/Program.cs

```diff
@@ -11,7 +11,39 @@ class Program
     {
         static void Main(string[] args)
         {
-            
+            Console.WriteLine("Choose an action:" +
+                "\n1.Create file" +
+                "\n2.Open file");
+            int actionStatus = Int32.Parse(Console.ReadLine());
+            switch(actionStatus)
+            {
+                case 1:
+                    Functionality.CreateFile();
```

Comment:
It looks quite weird when your " Functionality" class has methods for file, and "FileChanges" class has methods related more to real functionality